### ISSUE TITLE
python-for-android fails to write project.properties if the dist name has a space

### ISSUE THREAD

**Issue by inclement** [kivy/python-for-android#1079]
Reported at https://github.com/kivy/buildozer/issues/531#issuecomment-316155350

Not sure if the fix is to make the space work, or forbid spaces in the dist name (this would be necessary if there are problems with the java build tools or something).

**Comment by KeyWeeUsr** [comment]
Maybe just limit the dist name to `a-z + A-Z` and don't care about it. I doubt it's recommended to use spaces in such stuff (`_` instead) and it'd be even a safety-switch against non-ascii characters.

**Comment by inclement** [comment]
I don't think we should forbid things if there isn't a technical reason for it, especially not forbidding non-ascii characters.
